permissions/plugin-flock.yml

```diff
@@ -0,0 +1,7 @@
+---
+name: "flock"
+github: "jenkinsci/flock-plugin"
+paths:
+- "org/jenkins-ci/plugins/flock"
```

Comment:
Your path needs to use the groupId for your project, not the parent pom. 

https://github.com/jenkinsci/flock-plugin/blob/master/pom.xml#L12

So, your path would be io/jenkins/plugins/flock

Reply:
Thank you for providing the feedback on this, I've pushed again with the changes.